templates/annotate.sh

```diff
@@ -124,6 +127,72 @@ vep () {
   !{singularity_vep} vep "${args[@]}"
 }
 
+capice () {
```

Comment:
how much additional runtime does this step occur? test with Slurm on a WGS to verify that it stays within the maximum allowed annotation time.